packages/create-strapi-starter/create-strapi-starter.js

```diff
@@ -23,16 +28,58 @@ program
   .option('--dbauth <dbauth>', 'Authentication Database')
   .option('--dbfile <dbfile>', 'Database file path for sqlite')
   .option('--dbforce', 'Overwrite database content if any')
-  .description('create a new strapi starter application')
+  .description('Create a new strapi starter application')
   .action((directory, starterUrl, program) => {
     const projectArgs = { projectName: directory, starterUrl };
 
     buildStarter(projectArgs, program);
   });
 
-// Show help if not enough arguments are present
-if (process.argv.length < 4) {
-  program.help();
+// Get all possible options
```

Comment:
I'm not sure I undeerstand why you needed to do that ? 
It seems commander already supports requred arguments & logs for errors

Reply:
I did that because the way I was checking `process.argv.length` would  throw an error but not display the help (since the option would be included in the argv array).  If for example I did `create-strapi-starter --quickstart` project It would only return  `error: missing required argument <starterurl>`.  Maybe there is a better solution?

Reply:
never mind I think I found a better way 🤦